tensorflow/core/lib/io/buffered_inputstream.cc

```diff
@@ -157,14 +157,19 @@ Status BufferedInputStream::Seek(int64 position) {
   }
 
   // Position of the buffer within file.
```

Comment:
Let's not call this bufpos anymore because it is not.  E.g. rename to buf_lower_limit.

Also update the comment.

Reply:
thanks, updated